swap/src/network/request_response.rs

```diff
@@ -40,7 +40,7 @@ pub enum AliceToBob {
     Amounts(SwapAmounts),
     Message0(alice::Message0),
     Message1(alice::Message1),
-    Message2(alice::Message2),
+    Message2, // empty response
```

Comment:
Why responding if it's empty?

Reply:
This is how I understand it: each message exchange is part of a request-response protocol, always initiated by Bob (since Alice is the owner of the known address). I would assume that you could just omit empty responses, but then the individual request-response protocol would be unfinished.